src/explorer/SiteTreeItem.ts

```diff
@@ -170,6 +175,29 @@ export abstract class SiteTreeItem implements IAzureParentTreeItem {
             workspace.getConfiguration(constants.extensionPrefix, Uri.file(fsPath)).update(constants.configurationSettings.showBuildDuringDeployPrompt, false);
         }
     }
+
+    private async promptToSaveDeployConfigs(node: IAzureNode<SiteTreeItem>, fsPath: string): Promise<void> {
+        const saveDeploymentConfig: string = `Always deploy the workspace "${fsPath}" to "${node.treeItem.client.fullName}"?`;
```

Comment:
I would change this wording to "Default to "${node.treeItem.client.fullName}" for future deployments in this workspace?"

Reply:
This phrasing comes directly from big boss man @fiveisprime, but I have no real preference on how we want to express this.